src/ui/theme/recipes/SwitchRecipe.ts

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import { defineSlotRecipe } from '@chakra-ui/react';
+
+export const switchSlotRecipe = defineSlotRecipe({
+  slots: ['root', 'label', 'control', 'thumb', 'indicator'],
+  className: 'chakra-switch',
+  base: {
+    root: {},
+    thumb: {
+      bg: { base: '{colors.neutral.white}', _dark: '{colors.neutral.black}' },
```

Comment:
thoughts on using semantic tokens in recipes to make it more resilient for future changes?